acl.go

```diff
@@ -32,27 +32,17 @@ var (
 	}
 )
 
-func (c *Controller) HasRPCAccess(did, command string) bool {
+func HasRPCAccess(command string, mode AccessMode) bool {
 	switch command {
-	case "bitcoind":
-		return true
+	case "bind", "bind_ack", "bitcoind":
+		return mode > AccessModeNotApplicant && mode <= AccessModeMinimal
```

Comment:
Using comparison is implicit and causes some confusion and if someone adds incorrect `AccessMode` in between, the access control would be failed.

I think a better way would be using bitwise operations or create a list of allowance.